### DISCUSSION TITLE
how to import HTML into a RichTextField?

### DISCUSSION THREAD

**Discussion by mjmare** [wagtail/wagtail#10726] [General]
I see in the docs (https://docs.wagtail.org/en/stable/extending/rich_text_internals.html) that the rich text field has its own data format. What is the Wagtail way to import HTML into a Richtext field? 

TIA

**Comment by lb-** [comment]
It's mostly HTML, this section of that page is more helpful. 

https://docs.wagtail.org/en/stable/extending/rich_text_internals.html#data-format

Links and images will be the most different, along with some html simply not supported (such as IFrame/script). 

What kind of content is your source?

**Reply by mjmare** [reply]
The source is from a Django Summernote (https://github.com/summernote/django-summernote) field. The HTML shouldn't be too wild.
Being lazy, I was hoping for some method that would import regular HTML into the Richtext field...

**Reply by lb-** [reply]
Yeah, it's just a html string. Sorry that wasn't clear, just update it with the html from your source, do some testing, should get you close enough for something simple.

**Reply by mjmare** [reply]
So maybe the images need some fiddling, but the rest should be OK. Thanks!
